out_writeapi.go

```diff
@@ -256,8 +279,10 @@ func FLBPluginInit(plugin unsafe.Pointer) int {
 	configMap[counter] = &config
 
 	// Creating FLB context for each output, enables multiinstancing
+	config.mu.Lock()
```

Comment:
Not sure why these mutex changes show up here, as they are part of a separate checkin. You may have to rebase.